.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  Linting:
+    name: Linting NodeJS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
```

Comment:
```suggestion
        uses: actions/checkout@v3
```

Reply:
Idk from which repo you copied this but your local copy of that repo is outdated because I changed all of them.